test/integration/schedule/part2/index.test.js

```diff
@@ -6,7 +6,7 @@ describe('schedule part 2', () => {
   describe('Get top-up description', () => {
```

Comment:
Can you update the folder/file name to match the path under the app folder

Reply:
Would like your & Anwar's/Edi's opinion on this, how should we be testing this for this file.

We have fantastic unit tests for the 2 potential funcs it calls `getTopUpDescription` and `getReductionDescription`. But the tests in this are an inferior copy-paste from these meaning:
1. it's not testing anything new
2. we need to change tests in 2 places if the output of those funcs change

I guess what we're wanting to test is that the right func is called in the correct scenario and the output is as we're expecting.

Just questioning the best way of doing that.

Reply:
Your conclusion that the content of the two test files are the same is wrong. End of first paragraph of  `getTopUpDescription` test reads increased while that of `getReductionDescription` reads decreased.

The standard practice is to create test file for each function file. Are you suggesting we keep the functions together?
If your answer is yes, then we are deviating from the original plan of separating Top-up format implementation from that of reduction.  We agreed on separating them so that the two format are not tightly coupled, hence, implementation of any required change in one of the format can easily be achieved without much impact on the other.

Reply:
No, not suggesting moving the funcs together.

Reply:
Sorry, didn't mean to suggest content of each func is the same, Ik they're not.

Saying that we have unit tests checking the content for each func. This func is deciding which func to use and returning that. So the tests for this file should just be on the decision aspect not the content itself because that's covered in the lower unit tests.

Reply:
It has been resolved, contents are not the same.